components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -908,5 +910,90 @@ namespace scream {
             return band_index.hostRead();
         }
 
+        void compute_aerocom_cloudtop(
+            int ncol, int nlay, const real2d &tmid, const real2d &pmid,
+            const real2d &p_del, const real2d &z_del, const real2d &qc,
+            const real2d &qi, const real2d &cldfrac_tot, const real2d &nc,
+            real1d &T_mid_at_cldtop, real1d &p_mid_at_cldtop,
+            real1d &cldfrac_ice_at_cldtop, real1d &cldfrac_liq_at_cldtop,
+            real1d &cldfrac_tot_at_cldtop, real1d &cdnc_at_cldtop) {
+          /* The goal of this routine is to calculate properties at cloud top
+           * based on the AeroCOM recommendation. See reference for routine
+           * get_subcolumn_mask above, where equation 14 is used for the
+           * maximum-random overlap assumption for subcolumn generation. We use
+           * equation 13, the column counterpart.
```

Comment:
Thanks for this reference, that's really helpful. I think this is what I missing last time looking at this.